client/packages/system/src/Asset/ListView/ListView.tsx

```diff
@@ -40,7 +40,7 @@ const AssetListComponent: FC = () => {
       {
         key: 'manufacturer',
         Cell: TooltipTextCell,
-        maxWidth: 350,
+        width: 350,
```

Comment:
to stop the columns moving around when you scroll (due to the ViewPort and changing data values in the rows)